mars-form/src/components/Form.js

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import countries from "./country";
+
+
+class Form extends React.Component {
+  state = {
+    name : "",
+    dateofbirth : "yyyy-MM-dd",
+    country: "",
+    dietary:"",
+    marsResponse:"",
+    formCompleted: false,
+    formSubmitted: true
+  }
+  createOptions = () => {
+
+    return countries.map(country=>{
+      return(
+        <option value ={country.name} key={country.code} id={country.code}>{country.name}</option>
+
+      )
+    })
+  }
+  checkCompleted = (e) => {
+    e.preventDefault()
+
```

Comment:
destructor these for better practice. 
const {name, dateofbirth, country, marsResponse}  = this.state 
Also wouldn't mind seeing a helper lis if(this.hasAllProperites()) { do the thing) instead of doming all the chaining in an if.